### PR TITLE
Return more status when describe helm clusters

### PR DESCRIPTION
**Pull request by Hevienz**

### CODE REVIEW

File: pkg/db/schema/cluster/V0_10__cluster_node_host_ip.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE cluster_node
```
Comment (chilianyi) [inline pkg/db/schema/cluster/V0_10__cluster_node_host_ip.sql]: How about put these alter into one version file

File: pkg/plugins/helm/kubernetes_handler.go [hunk 2]
```diff
@@ -141,70 +140,19 @@ func (p *KubeHandler) WaitPodsRunning(runtimeId, namespace string, clusterRoles
 	return err
 }
 
-func (p *KubeHandler) getPodsByClusterRole(namespace string, clusterRole *models.ClusterRole) (pods *corev1.PodList, err error) {
-	kubeClient, _, err := p.initKubeClient()
+func (p *KubeHandler) GetClusterRolesWithStatusAndClusterNodes(cluster *models.Cluster, clusterRoles *map[string]*models.ClusterRole) (map[string]*models.ClusterNodeWithKeyPairs, error) {
```
Comment (chilianyi) [inline pkg/plugins/helm/kubernetes_handler.go]: The func name is too long and confused

File: pkg/db/schema/cluster/V0_10__cluster_node_host_ip.sql (hunk 1 above)
Comment (Hevienz) [reply pkg/db/schema/cluster/V0_10__cluster_node_host_ip.sql]: ok

File: api/cluster.proto [hunk 3]
```diff
@@ -233,9 +233,11 @@ message ClusterNode {
 	google.protobuf.BoolValue auto_backup = 21;
 	google.protobuf.Timestamp create_time = 22;
 	google.protobuf.Timestamp status_time = 23;
-	ClusterRole cluster_role = 24;
-	ClusterCommon cluster_common = 25;
-	repeated string key_pair_id = 26;
+	google.protobuf.StringValue host_name = 24;
```
Comment (zheng1) [inline api/cluster.proto]: host_name => hostname

Comment (Hevienz) [reply api/cluster.proto]: ok

**Comment by chilianyi** [comment]
LGTM

**Pull request merged**